### PR TITLE
fix(theme): make links more readable

### LINKED ISSUE #119: Links in dark theme are not readable
**To Reproduce:**
Profilbild > Einstellungen > Farbschema: Blau-Dunkel Farbschema
Beobachtung:
Links auf sind dunkelblau auf dunkelgrauem Hintergrund. Das ist sehr schwer lesbar.

**Expected behavior:**

Bei dem Farbschema sind Links als selbige zu erkennen und gut lesbar.

**Screenshots:**

image

**Additional Context**
Internal Issue: 8728

### PR DESCRIPTION
**Pull request by DanielHabenicht**
fixes #119

### CODE REVIEW

**Comment by DanielHabenicht** [comment]
What do you guys think? Is it ok to change the default magenta? Or should we use the secondary color for links?

**Comment by mschwrdtnr** [comment]
I think that this is a good solution and I would leave it like that. But I am not really sure if the colors are a bit to pale (in view of accessibility..).

**Comment by DanielHabenicht** [comment]
so you mean it is still not enough contrast?

**Comment by mschwrdtnr** [comment]
I would they it's enough but 
image

**Comment by DanielHabenicht** [comment]
Mhm, dann ist aber der Unterschied zwischen den Body Text und dem Link nicht mehr so gut erkennbar. 

Ich würde es jetzt erstmal so durchmergen. Es gibt ja noch #4

**Comment by Zeroks77** [comment]
image

found this.... 

pls fix it

**Comment by mschwrdtnr** [comment]
@Zeroks77 good point! And same for me with magenta.
image

**Comment by DanielHabenicht** [comment]
well, it's also that bad in the angular material demo:
image
https://material.angular.io/components/select/overview

**Comment by DanielHabenicht** [comment]
How about chips?
image
but the tables does not look so good:
image

**Comment by mschwrdtnr** [comment]
I think the idea is very cool. Maybe you can change the color of the chips to a bright magenta or something like? Could look cool.

But the problem with the tables is not acceptable. Also haven't a good idea to make it better.

**Comment by DanielHabenicht** [comment]
Fixed the table problem: 
image

But it does not look good with too much color:
image

**Comment by mschwrdtnr** [comment]
I don't really like this. What to you think?
image

**Comment by DanielHabenicht** [comment]
It just depends on the width of the table and how many columns there are. For my config every thing is fine: 
image

**Comment by DanielHabenicht** [comment]
Before it was just cut off:
image

**Comment by mschwrdtnr** [comment]
Okay. I would say use the chips and the table layout should be fixed with #452 
What are you guys say @T-Systems-MMS/phonebook-developers ?

For my opinion it does not look that bad
image
It's a black design and this can be a bit spacy :D

I really like the idea of using chips for everything you can filter for and it makes sense that the clicked chip is pasted to the searchbar.

**Comment by DanielHabenicht** [comment]
I don't like so much color. I think they are visible enough as chips. 
Also, the material design guidelines say that primary and accent colored chips are for selected or activated chips only.

**Comment by mschwrdtnr** [comment]
> I don't like so much color, they draw to much attention. I think they are visible enough as chips.
> Also, the material design guidelines say that primary and accent colored chips are for selected or activated chips only.

My only "problem" is that the whole page except the navbar ist black/white... but it's acceptable for me.

**Comment by DanielHabenicht** [comment]
We can speak about it in the redesign tickets, like #106

**Comment by mschwrdtnr** [comment]
There is still the problem with the visibility @Zeroks77 described..

**Comment by DanielHabenicht** [comment]
Thats for another PR. This does only targets links.

File: Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html [hunk 1]
```diff
@@ -233,15 +233,15 @@ <h4 mat-line>
               </a>
             </h4>
             <p mat-line>
-              <li *ngFor="let shortOrgUnit of person.Business.ShortOrgUnit; let index = index">
+              <mat-chip-list class="mat-chip-list-stacked">
```
Comment (mschwrdtnr) [inline Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html]: Where is the class `mat-chip-list-stacked` defined?

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html]: it comes with material

Comment (mschwrdtnr) [reply Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html]: Where can i find this classes? Didn't saw it here https://material.angular.io/components/chips/api

Comment (DanielHabenicht) [reply Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html]: Right there: https://material.angular.io/components/chips/overview#orientation

**Pull request merged**